calico/reference/resources/bgppeer.md

```diff
@@ -34,18 +34,19 @@ spec:
+| sourceAddress            | Specifies whether and how to configure a source address for the peerings generated by this BGPPeer resource.  Default value "UseNodeIP" means to configure the node IP as the source address.  "None" means not to configure a source address.                                                                                                                                                     | "UseNodeIP", "None"                                                                         | string                            | "UseNodeIP"                                                       |
+| maxRestartTime           | Restart time that is announced by BIRD in the BGP graceful restart capability and that specifies how long the neighbor would wait for the BGP session to re-establish after a restart before deleting stale routes. Note: extra care should be taken when changing this configuration, as it may break networking in your cluster. When not specified, BIRD uses the default value of 120 seconds. | `10s`, `120s`, `2m` etc.                                                                    | [Duration string][parse-duration] | `nil` (empty config, BIRD will use the default value of `120s`)   |
+| numAllowedLocalASNumbers | The number of local AS numbers to allow in the AS path for received routes. This disables BGP loop prevention and should only be used if necessary.                                                                                                                                                                                                                                                |                                                                                             | integer                           | `nil` (BIRD will default to 0 meaning no change to loop prevention behavior) |
+| ttlSecurity              | Enables the generalized TTL security mechanism (GTSM) which protects against spoofed packets by ignoring received packets with a smaller than expected TTL value. The provided value is the number of hops (edges) between the peers. Does not apply to mesh peers.                                                                                                                                | 0 - 255                                                                                     | 8-bt integer                      | `nil` (results in BIRD configuration `ttl security off`)          |
```

Comment:
I don't think you need the "does not apply to mesh peers" bit here - the fact that this is on a BGPPeer object means it could never apply to mesh peers anyway.

Reply:
Ah that's true. I've removed it.